github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```
@@ -616,6 +616,22 @@ presets:
   volumeMounts:
   - name: bazeluserroot
     mountPath: /tmp/cache/bazel
+- labels:
+    preset-kubevirt-icr-credential: "true"
```

probably worth pointing out that this is only available on the s390x bare-metal cluster
```suggestion
    preset-kubevirt-s390x-icr-credential: "true"
```

I don't see this preset being used by any other lanes outside of s390x

---

done